TAC/Meetings/2024/2024-06-13/CCC-TAC-Minutes-2024-06-13

```diff
@@ -0,0 +1,114 @@
+* **LF Training course on DEI**: [Inclusive Open Source Community Orientation (LFC102) (free)](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)
+* **Declared project dependencies**: [Google Sheets](https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit#gid=0)
+* **CCC YouTube**: [youtube.confidentialcomputing.io](https://youtube.confidentialcomputing.io)
+* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
+* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
+* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)
+
+
+#### Voting members of the TAC
+
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+* AMD - David Kaplan / Harold Gilkey +
+* Arm - Nathaniel McCallum   / Michael Lu
+* Google - Catherine Zhang +
+* Huawei - Zhipeng (Howard) Huang 
+* Intel - Dan Middleton (TAC chair)   / Simon Johnson
+* Meta -  Henry Wang + / * Kevin Hui +
+* Microsoft - Alec Fernandez 
+* Nvidia - Fritz Alder +
+* Red Hat - Yash Mankad + / Ram Pai +
+* TikTok - Mingshen Sun   / Yao Zhang
+
+   " + " indicates attending
+
+#### Other Attendees
+
+* Dionna Glaze (Google)
+* Harold Gilkey (AMD)
+* Hugo Vincent (Arm Limited)
+* John Manferdelli
+* Kanth Ghatraju (Oracle)
+* Manu Foutaine (Hushmesh)
+* Ofir Azoulay-Rozanes (Anjuna)
+* Ram Pai (IBM)
+* Ye Li
+
+* Mike Bursell (LF)
+* Riaan Kleinhans (LF)
+* Ben Sternthal (BS)
+
+
+## Welcome
+
+Dan Middleton (DM) opened the call at 7:09 am and reviewed the antitrust policy. 
+
+## Agenda 
+
+DM reviewed the agenda for the meeting. 
+
+## Roll Call / Attendance
+
+Since no votes would be occurring in the meeting, attendance was skipped. 
+
+## Welcome New Community Members
+
+*  Hugo Vincent from ARM introduced themselves.
+
+## Announcements
+
+Riaan Kleinhans (RK) provided an update on the brand repositioning working group, and pointed interested people towards a slack channel. DM stated that he had spoken to Kate and the outreach committee about this workgroup, and they noted it would benefit from TAC representation.
+
+DM called attention to the recent NIST publication. 
+
+
+## CC Summit Feedback
+
+DM asked for feedback on the recent CC summit conference. Ofir Azoulay-Rozanes (OAR) stated that it was the first conference he attended where people understood confidential computing. OAR mentioned that AI was a major topic of discussion, but he felt it was important for people to understand that confidential computing is not just for AI—it is important for any application with sensitive data.
+Fritz Alder said he found the discussion balance between AI solutions to come and current solutions that are available.
```

Comment:
Minor tweak, but as written I did not understand this sentence.

Fritz Alder said he found the discussion balance**d** between future AI solutions and current available solutions.

Reply:
Suggest rephrasing to: "... balanced between AI solutions to come and CC product offerings that are available today"

Reply:
Corrected with @cxzhang suggestion